bedrock/firefox/templates/firefox/quantum.html

```diff
@@ -0,0 +1,190 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-pebbles.html" %}
+
+{% block page_css %}
+  {% stylesheet 'firefox-quantum' %}
+{% endblock %}
+
+{% block page_title %}{{ _('2x faster and 30% less memory') }} | {{ _('Firefox Quantum Browser') }}{% endblock %}
+{% block page_desc %}{{ _('The new Firefox arrives November 14, 2017') }}{% endblock %}
+
```

Comment:
Almost forgot, we should add the new meta page image here.

Reply:
Will do 👍